srv/components/ha/corosync-pacemaker/reset.sls

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+Remove user and group:
+  user.absent:
+    - name: hacluster
+    - purge: True
+    - force: True
+
+{% for serv in ["corosync", "pacemaker", "pcsd"] %}
+Stop service {{ serv }}:
+  service.dead:
+    - name: {{ serv }}
+    - enable: False
+{% endfor %}
+
+Remove pcs package:
+  cmd.run:
+    - name: "rpm -e --nodeps pcs pacemaker corosync fence-agents-ipmilan"
+
+# Remove configuration directory:
```

Comment:
Please confirm if we have to remove config directory for services
`"corosync", "pacemaker", "pcsd"`

Reply:
We are doing it as part of the same file.